<ui version="4.0" >
 <class>ConfigWindow</class>
 <widget class="QDialog" name="ConfigWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QuarkPlayer - Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item rowspan="2" row="0" column="0" >
    <widget class="QTreeWidget" name="treeWidget" >
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QStackedWidget" name="stackedWidget" />
   </item>
   <item row="1" column="1" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../quarkplayer-app/quarkplayer.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigWindow</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>547</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigWindow</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>582</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
